server/config/forms/useOfForceDetailsForm.js

```diff
@@ -52,7 +52,7 @@ const completeSchema = joi.object({
           .items(
             requiredOneOfMsg(
               'STANDING',
-              'FACE_DOWN',
+              'ON_FRONT',
```

Comment:
can't change this - it won't match whats in the DB

Reply:
thanks